src/main/java/com/jackshenorion/smarts/util/io/IOUtil.java

```diff
@@ -525,7 +525,9 @@ public static void writeAndClose(File file, byte[] data) {
 
```

Comment:
**Reporter**: FINDBUGS
**Rule**: NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE
**Severity**: WARN
**File**: src/main/java/com/jackshenorion/smarts/util/io/IOUtil.java L274
**Source**: com.jackshenorion.smarts.util.io.IOUtil

Possible null pointer dereference due to return value of called method


<p> The return value from a method is dereferenced without a null check,
and the return value of that method is one that should generally be checked
for null.  This may lead to a <code>NullPointerException</code> when the code is executed.
</p>
      

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<-1970672952>*